editor/export/project_export.cpp

```diff
@@ -51,6 +52,44 @@
 #include "scene/gui/texture_rect.h"
 #include "scene/gui/tree.h"
 
+void ProjectExportTextureFormatError::_on_fix_texture_format_pressed() {
+	ProjectSettings::get_singleton()->set_setting(setting_identifier, true);
+	ProjectSettings::get_singleton()->save();
+	EditorFileSystem::get_singleton()->scan_changes();
+	emit_signal("texture_format_enabled");
+}
+
+void ProjectExportTextureFormatError::_bind_methods() {
+	ADD_SIGNAL(MethodInfo("texture_format_enabled"));
+}
+
+void ProjectExportTextureFormatError::_notification(int p_what) {
+	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE:
```

Comment:
This was excessive btw, `NOTIFICATION_THEME_CHANGED` already fires when the node enters the tree.